pom.xml

```diff
@@ -193,4 +187,25 @@
         </pluginManagement>
     </build>
 
+    <profiles>
+        <profile>
+            <id>all</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>api</module>
+                <module>contract</module>
+                <module>e2e-tests</module>
+                <module>serde-api</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>contract</id>
```

Comment:
We don't quite need the `contract` profile. To achieve the same one can build just one module, `contract`.
What we need is a jar without the frontend, basically what skipUiBuild does now, but as a profile.

Reply:
should be solved by now